### PR TITLE
fix/tutorial link

### PR DESCRIPTION
**Pull request by just-at-uber**
Navigation was showing a link which the page is now removed. This PR removes this link and also updates folder numbering.

### Changed
- updated folder structure numbering

### Removed
- removed tutorial link from navigation

### Before
Screen Shot 2021-04-02 at 9 33 10 AM
Screen Shot 2021-04-02 at 9 35 22 AM

### After
Screen Shot 2021-04-02 at 9 33 17 AM

### CODE REVIEW

File: src/.vuepress/config.js [hunk 1]
```diff
@@ -35,7 +35,6 @@ module.exports = {
           { text: 'Get Started', link: '/docs/get-started/' },
           { text: 'Use cases', link: '/docs/use-cases/' },
           { text: 'Concepts', link: '/docs/concepts/' },
-          { text: 'Tutorials', link: '/docs/tutorials/' },
```
Comment (just-at-uber) [inline src/.vuepress/config.js:38]: > - removed tutorial link from navigation

File: src/.vuepress/config.js [hunk 2]
```diff
@@ -85,13 +84,13 @@ module.exports = {
       '/docs/': [
         {
           title: 'Get Started',
-          path: '/docs/00-get-started',
+          path: '/docs/01-get-started',
```
Comment (just-at-uber) [inline src/.vuepress/config.js:87]: > - updated folder structure numbering

File: src/.vuepress/config.js (hunk 1 above)
Comment (mkolodezny) [inline src/.vuepress/config.js:38]: did we replace tutorials by something else?

**Review by mkolodezny — APPROVED** [review]
LGTM

File: src/.vuepress/config.js (hunk 1 above)
Comment (just-at-uber) [reply src/.vuepress/config.js:38]: It was moved into the getting started section awhile ago. This was just not cleaned up.

**Pull request merged**